modal/_functions.py

```diff
@@ -1667,7 +1668,7 @@ async def from_id(
         return fc
 
     @staticmethod
-    async def gather(*function_calls: "_FunctionCall[Any]") -> list[Any]:
+    async def gather(*function_calls: "_FunctionCall[T]") -> typing.Sequence[T]:
```

Comment:
Couldn't use `ReturnType` here since this is a staticmethod.

Reply:
Ah, yeah that makes sense, since ReturnType is a generic in the class